urllib3/exceptions.py

```diff
@@ -176,3 +176,10 @@ class ProxySchemeUnknown(AssertionError, ValueError):
     def __init__(self, scheme):
         message = "Not supported proxy scheme %s" % scheme
         super(ProxySchemeUnknown, self).__init__(message)
+
+
+class HeaderParsingErrors(HTTPError):
+    def __init__(self, defects, unparsed_data):
+        message = '%s, unparsed data: %r' % (
```

Comment:
Still weren't able to assign the properties? This is fine though.

Also such col80, very wow.